docs/babelrc.md

```diff
@@ -3,67 +3,17 @@ title: .babelrc
 id: babelrc
 ---
 
-Babel can be configured! Many other tools have similar configs: ESLint (`.eslintrc`), Prettier (`.prettierrc`).
-
-All Babel API [options](core.md#options) are allowed. However if the option requires JavaScript, you may need to use a `.babelrc.js` file.
-
-## Config types
-
-- `.babelrc`
-- `.babelrc.js`
-- `package.json` key
-- Tools (`@babel/core`, `@babel/register`) may take in options directly
-- `@babel/cli`
-
-## Use via `.babelrc`
-
-```json
-{
-  "plugins": ["@babel/plugin-transform-arrow-functions"]
-}
-```
-
-## Use via `.babelrc.js`
-
-```json
-module.exports = {
-  "plugins": ["@babel/plugin-transform-arrow-functions"]
-};
-```
-
-## Use via `package.json`
-
-Alternatively, you can choose to specify your `.babelrc` config from within `package.json` like so:
-
-```json
-{
-  "name": "my-package",
-  "version": "1.0.0",
-  "babel": {
-    // my babel config here
-  }
-}
-```
-
-## Use via `@babel/core`
-
-```js
-require("@babel/core").transform("code", {
-  plugins: ["@babel/plugin-transform-arrow-functions"]
-});
-```
+## Lookup behavior
 
-## Use via `@babel/cli`
+Babel will look for a `.babelrc` in the current directory (which defaults to cwd).
```

Comment:
For lookup I thought you would want to explain how it keeps looking up the folder until it hits package.json?

Reply:
According to the pr (https://github.com/babel/babel/pull/7358) it doesn't do that anymore? See "When searching for .babelrc files for a given file, stop at package.json"

Reply:
Oh Yeah that's what I meant, about it stopping at package.json?